adapters/Vungle/VungleAdapter/GADMAdapterVungleInterstitial.m

```diff
@@ -40,6 +40,10 @@ @implementation GADMAdapterVungleInterstitial {
   return [VungleAdNetworkExtras class];
 }
 
++ (NSString *)adapterVersion {
+  return kGADMAdapterVungleVersion;
+}
```

Comment:
Add this method to fix the warning " Class 'GADMAdapterVungleInterstitial' does not conform to protocol 'GADMAdNetworkAdapter'"